controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/Maintainer.java

```diff
@@ -116,7 +116,7 @@ static long staggeredDelay(List<HostName> cluster, HostName host, Instant now, D
 
         long offset = cluster.indexOf(host) * interval.toMillis() / cluster.size();
         long timeUntilNextRun = Math.floorMod(offset - now.toEpochMilli(), interval.toMillis());
-        return timeUntilNextRun + interval.toMillis() / cluster.size();
+        return timeUntilNextRun;
```

Comment:
Should be changed in `com.yahoo.vespa.hosted.provision.maintenance.Maintainer` too.